setup.py

```diff
@@ -105,17 +105,17 @@ def check_setuptools_features():
     url='https://github.com/BigchainDB/bigchaindb/',
     author='BigchainDB Contributors',
     author_email='dev@bigchaindb.com',
-    license='AGPLv3',
+    license='Apache Software License 2.0',
     zip_safe=False,
 
     classifiers=[
-        'Development Status :: 4 - Beta',
+        'Development Status :: 5 - Production/Stable',
```

Comment:
How would this work when we release an `rc`? To me, we should keep `4 - Beta` for now.

Reply:
Let's discuss Thursday.

Reply:
See below: I changed the dev status back to "3 - Alpha" for now.